src/main/java/org/spongepowered/asm/mixin/transformer/MixinTargetContext.java

```diff
@@ -1499,6 +1500,17 @@ void applyInjections(int injectorOrder) {
             throw new InvalidMixinException(this, "Unexpecteded " + ex.getClass().getSimpleName() + " whilst transforming the mixin class:", ex,
                     this.activities);
         }
+
+        //Strip synthetic local variables from the LVT to avoid debuggers becoming confused by them
```

Comment:
This was deliberately placed after the class export before so the dump would still have local names. That behaviour should be preserved.

Reply:
I will have to lift it into a separate method in `MixinTargetContext`, then call it after the check for any suppressed exceptions:
https://github.com/FabricMC/Mixin/blob/26c8d9c12b1ffa8d1f7cc1e42acb5b113f192b30/src/main/java/org/spongepowered/asm/mixin/transformer/MixinProcessor.java#L362-L376

Reply:
With the custom interface, `ExtensionLVTCleaner` has been put back, as it can now check whether the LVT could have been modified before needing the `ClassNode`